tests/robot/suites/trafficIPv6/tap_int_traffic/tap_int_trafficIPv6.robot

```diff
@@ -54,7 +55,7 @@ Add VPP1_TAP1 Interface
 
 Check VPP1_TAP1 Interface Is Created
     ${interfaces}=       vat_term: Interfaces Dump    node=agent_vpp_1
-    vpp_term: Interface Is Created    node=agent_vpp_1    mac=${MAC_VPP1_TAP1}
+    Wait Until Keyword Succeeds   ${WAIT_TIMEOUT}   ${SYNC_SLEEP}    vpp_term: Interface Is Created    node=agent_vpp_1    mac=${MAC_VPP1_TAP1}
```

Comment:
Keyword `vpp_term: Interface Is Created` already contains `Wait Until Keyword Succeeds`. We certainly don't want to plague every level with this..